app/src/main/java/net/squanchy/notification/NotificationService.java

```diff
@@ -6,64 +6,58 @@
 import android.content.Context;
 import android.content.Intent;
 
-import java.util.Calendar;
 import java.util.Comparator;
-import java.util.Date;
 
 import net.squanchy.schedule.domain.view.Event;
 
+import org.joda.time.DateTime;
+import org.joda.time.Minutes;
+
 import io.reactivex.Observable;
 
 public class NotificationService extends IntentService {
 
-    private static final int NOTIFICATION_INTERVAL_MINUTES = 10;
+    private static final Minutes NOTIFICATION_INTERVAL = Minutes.minutes(10);
+    
+    private final NotificationCreator notificationCreator;
+    private final Notifier notifier;
 
     public NotificationService() {
         super(NotificationService.class.getSimpleName());
+        notificationCreator = new NotificationCreator(this);
+        notifier = Notifier.from(this);
     }
 
     @Override
     protected void onHandleIntent(Intent intent) {
-        NotificationCreator notificationCreator = new NotificationCreator(this);
-        Notifier notifier = Notifier.from(this);
-
-        Calendar calendar = Calendar.getInstance();
-        calendar.add(Calendar.MINUTE, NOTIFICATION_INTERVAL_MINUTES);
-        Date notificationIntervalEnd = calendar.getTime();
-        Date now = new Date();
+        DateTime notificationIntervalEnd = new DateTime().withPeriodAdded(NOTIFICATION_INTERVAL, 1);
```

Comment:
Why using the Period thing with a count of one and not `plus(NOTIFICATION_INTERVAL)`?

PS: you probably don't want to use a `ReadablePeriod` such as `Minutes` when adding, according to the `plus()` docs:

>This method is typically used to add complex period instances. Adding one field is best achieved using methods like `#plusYears(int)`.

You can use `plusMinutes()` and leave the constant being an `int`